src/lib/utils/translations.ts

```diff
@@ -58,6 +58,10 @@ const getRequiredNamespacesForPath = (path: string) => {
     requiredNamespaces = [...requiredNamespaces, "page-community"]
   }
 
+  if (path.startsWith("/dapps")) {
+    requiredNamespaces = [...requiredNamespaces, "page-dapps", "glossary"]
```

Comment:
We could put "glossary" into its own condition, similar to what we did with the "learn-quizzes" since it is needed in many pages.

```ts
if (path.startsWith("/dapps") || ...) {
  requiredNamespaces = [...requiredNamespaces, "glossary"]
}
```